go/libkb/pgp_key.go

```diff
@@ -607,6 +609,13 @@ func (k *PGPKeyBundle) CheckFingerprint(fp *PGPFingerprint) error {
 }
 
 func (k *PGPKeyBundle) SignToString(msg []byte) (sig string, id keybase1.SigID, err error) {
+	if k.Entity.PrivateKey == nil || k.Entity.PrivateKey.PrivateKey == nil {
```

Comment:
Another idea is to make a new key type that is a PGPKey with a GPG-fallback, and explicit use that key in the case of `GPGImportKeyEngine` and a "pregen" key?  That would be a safe thing to do, to only enable it in the two cases: login and `pgp select`.

@jacobhaven which cases do you think we'd cut out with this (conservative) approach that you think might be useful?